Makefile

```diff
@@ -20,7 +20,11 @@ profile:
 	@sudo cp profile.sh /etc/profile.d/lead-workspace.sh
 	
 microk8s:
-	sudo snap install kubectl --classic --channel=${K8S_VERSION}
+	sudo apt-get update && sudo apt-get install -y apt-transport-https
+	curl -s https://packages.cloud.google.com/apt/doc/apt-key.gpg | sudo apt-key add -
+	echo "deb https://apt.kubernetes.io/ kubernetes-xenial main" | sudo tee -a /etc/apt/sources.list.d/kubernetes.list
+	sudo apt-get update
+	sudo apt-get install -y kubectl
```

Comment:
is there a way to pin the kubectl version?

Reply:
I'll do that first thing tomorrow, if I can't find a way via apt-get then I'll just pull the binary similar to how I do it for helm and hub

Reply:
Updated it to download the binary instead 👍